airbyte-cdk/java/airbyte-cdk/typing-deduping/src/main/java/io/airbyte/integrations/base/destination/typing_deduping/FutureUtils.java

```diff
@@ -12,29 +12,40 @@
 
 public class FutureUtils {
 
+  public static final int DEFAULT_TD_THREAD_COUNT = 8;
```

Comment:
if you want the caller to interact only through provided getter, make it private.